includes/class-wc-tracker.php

```diff
@@ -321,6 +339,41 @@ private static function get_orders() {
 		return array_merge( $order_dates, $order_counts, $order_totals );
 	}
 
+	/**
+	 * Get review counts for different statuses.
+	 *
+	 * @return array
+	 */
+	private static function get_review_counts() {
+		global $wpdb;
+		$review_count = array();
+		$counts       = $wpdb->get_results( "
+			SELECT comment_approved, COUNT(*) AS num_reviews
+			FROM {$wpdb->comments}
+			WHERE comment_type = 'review'
```

Comment:
Just a heads-up a lot of comments don't currently have the 'review' comment type (See #21305). There should be an update script that runs, but data for review counts may not be reliable at first.